_posts/2019-04-24-gis-wisdom.md

```diff
@@ -0,0 +1,111 @@
+
+In the versioning workflow, we eventually need to validate our version's edits and push (aka: post) them to the parent version so other editors can see and incorporate them.  This is called reconcile and post.  It is during this process where we may encounter data conflicts. 
+
+When we reconcile, we fetch what's new from the parent and resolve any differences we have with it.  It is only after we have resolved the differences that we can then post our edits to the parent version.  Once we post, our version is then safely validated and checked-in. At that point, we have an agreed-upon baseline for the data on which we can safely begin editing again. 
+
+It's not hard to imagine how we can incorporate some of these concepts into our life.  Think of how similar a version is to our worldview.  We often want to validate our views, so we engage in dialog with others.  Inevitably conflicts arise. Ideally, we should first attempt to fully understand the other viewpoint and then proceed with explaining our own.  It is from there that we can work out our differences, or kindly ‘agree to disagree’.  Understanding both viewpoints can help us establish grounds to move forward on.
+
+Learn more about the [ArcGIS versioning process](http://desktop.arcgis.com/en/arcmap/latest/manage-data/geodatabases/the-version-editing-process.htm).
+
+### 2. Networks 
+{: .text-left}
+
+`networking`
+
+In GIS, we build network datasets for navigation.  These networks are based on connectivity.  We use terms such as edges and junctions, but really we’re just establishing resource groups to which we grant connectivity to other sources.
+
+Likewise, building a personal network helps us determine who and what we allow ourselves to connect with.  A well-thought-out network not only tells us where we can connect but also where we should not. Taking the time to think about this connectivity ensures that we build a quality social network.
+
+Learn more about [creating an ArcGIS network datasets](http://desktop.arcgis.com/en/arcmap/latest/extensions/network-analyst/creating-a-network-dataset.htm) and [how connectivity works](http://desktop.arcgis.com/en/arcmap/latest/extensions/network-analyst/understanding-connectivity.htm).
+
+### 3. Extract, Translate, and Load (ETL) 
+{: .text-left}
+
+`being adaptable`
+
+You have probably heard the phrase, 'build once, use many'.  When building data, we say this when we want the output to be adaptable and used for many purposes.  Take road centerlines for example.  This dataset should be designed with many uses in mind (such as routing, mapping, locating, assessment management, etc.).  With a simple ‘extract, translate, and load’ (ETL), the data should meet the needs of many users.  
```

Comment:
`locating, assessment management, etc.).` did you mean assessment or possibly asset?